packages/redis_client/example/main.dart

```diff
@@ -9,24 +7,9 @@ Future<void> main() async {
   // Connect to the Redis server.
   await client.connect();
 
-  const key = 'HELLO';
-
-  final initialValue = await client.get(key: key); // null
-  assert(initialValue == null, 'Key should not exist.');
-
-  // Set the value of a key.
-  await client.set(key: key, value: 'WORLD');
-
-  // Get the value of a key.
-  final value = await client.get(key: key); // WORLD
-  assert(value == 'WORLD', 'Value should be "WORLD".');
-
-  // Delete the key.
-  await client.delete(key: key);
-
-  final finalValue = await client.get(key: key); // null
-  assert(finalValue == null, 'Key should not exist.');
+  // Execute a command.
+  await client.execute(['PING']); // PONG
 
```

Comment:
Why get rid of the get/set/delete examples?

Reply:
Just thought the simplest getting started would be best but happy to change it back 🤷